src/library/Box/Log.php

```diff
@@ -24,33 +24,32 @@
  */
 class Box_Log implements \Box\InjectionAwareInterface
 {
-    const EMERG   = 0;  // Emergency: system is unusable
-    const ALERT   = 1;  // Alert: action must be taken immediately
-    const CRIT    = 2;  // Critical: critical conditions
-    const ERR     = 3;  // Error: error conditions
-    const WARN    = 4;  // Warning: warning conditions
-    const NOTICE  = 5;  // Notice: normal but significant condition
-    const INFO    = 6;  // Informational: informational messages
-    const DEBUG   = 7;  // Debug: debug messages
-
-    protected $_priorities = array(
-         self::EMERG    => 'EMERG',
-         self::ALERT    => 'ALERT',
-         self::CRIT     => 'CRIT',
-         self::ERR      => 'ERR',
-         self::WARN     => 'WARN',
-         self::NOTICE   => 'NOTICE',
-         self::INFO     => 'INFO',
-         self::DEBUG    => 'DEBUG',
+    const EMERG = 0;  // Emergency: system is unusable
+    const ALERT = 1;  // Alert: action must be taken immediately
+    const CRIT = 2;  // Critical: critical conditions
+    const ERR = 3;  // Error: error conditions
+    const WARN = 4;  // Warning: warning conditions
+    const NOTICE = 5;  // Notice: normal but significant condition
+    const INFO = 6;  // Informational: informational messages
+    const DEBUG = 7;  // Debug: debug messages
+
+    protected array $_priorities = array(
```

Comment:
Very minor, but arrays can be declared using the [] syntax since PHP 5.4 which seems to now be preferred. 🙂